db/migrate/20220707204107_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :user_id, :integer
+  end
+end
```

Comment:
- [ ] Kindly note that this migration is not adding the user_id reference to the user table this migration is just only adding a column `user_id` to the post table without any reference. To make this reference kindly run this migration `rails g migration addUserToPosts user:references`, this will generate a migration like this

```rb
class AddUserToPosts < ActiveRecord::Migration[7.0]
  def change
    add_reference :posts, :user, null: false, foreign_key: true
  end
end
```